### ISSUE TITLE
Delete query results

### ISSUE THREAD

**Issue by JohnBruce** [jettro/elasticsearch-gui#79]
Unless I'm missing something, theres no way to delete the results of queries through the UI.  You could only do this by running a manual command.

This seems like a pretty powerful feature to have been left out so perhaps maybe its there and just not obvious?

**Comment by jettro** [comment]
You are right, the goal of this tool when I created it was to learn about the data and how the queries work. After that I started doing index maintenance. But it is a good idea and I will think about it.

**Comment by blainsmith** [comment]
Any more on this even tho this is a very old issue?

**Comment by jettro** [comment]
Since elasticsearch took out the API, this cannot be done easily. I would have to add scan/scroll functionality. It is certainly doable, but not on the top of my list at the moment.

> Deprecated in 1.5.3.
> "Delete by Query will be removed in 2.0: it is problematic since it silently forces a refresh which can quickly cause OutOfMemoryError during concurrent indexing, and can also cause primary and replica to become inconsistent. Instead, use the scroll/scan API to find all matching ids and then issue a bulk request to delete them..
